engine/execution/state/bootstrap/bootstrap.go

```diff
@@ -41,7 +41,7 @@ func (b *Bootstrapper) BootstrapLedger(
 
 	ctx := fvm.NewContext(b.logger, fvm.WithChain(chain))
 
-	err := vm.Run(ctx, fvm.Bootstrap(servicePublicKey, initialTokenSupply), view)
+	err := vm.Run(ctx, fvm.Bootstrap(servicePublicKey, fvm.WithInitialTokenSupply(initialTokenSupply)), view)
```

Comment:
Maybe put arguments on separate lines and/or introduce a variable for the `fvm.Bootstrap` expression, to make the multiple nested expressions more easy to read